vernac/vernacstate.mli

```diff
@@ -29,8 +17,7 @@ module Synterp : sig
   val init : unit -> t
   val freeze : unit -> t
   val unfreeze : t -> unit
-  val parsing : t -> Parser.t
-
+  val parsing : t -> Pcoq.frozen_t
```

Comment:
Is this still used?

Reply:
It is not but I kept it on purpose for now.

Reply:
I stand corrected, it is indeed actually used in the stm to recover the parsing state from the full state.